_events/sr2021/kickstart.md

```diff
@@ -1,13 +1,23 @@
 ---
 title: Kickstart
-date: 2020-11-21
+date: 2020-11-21 13:00:00
 layout: event
 type: kickstart
 location: The Student Robotics YouTube Channel
 ---
 
-Kickstart is the event which kicks off a competition year, where the game and rules will
-be announced and our competition simulator will shown off to teams.
+Kickstart is the event which kicks off a competition year. The game and rules will
+be announced, the competition simulator will be shown off to teams and of course any
+questions you have will be answered. Kickstart will be streamed live on
+**21st November 2020 at 13:00 (GMT)**, aiming to finish around 16:00.
+
+The simulator will be based in [Webots](https://cyberbotics.com/#download). The download
+is large, and we recommend downloading it before the event to ensure you can get started
+immediately. Information on downloading our simulation will be made available through
+[the docs](https://studentrobotics.org/docs/competition-simulator/) during the event.
+
+During the event, we recommend setting up a call or video chat for you and your team to
+communicate and collaborate, both to share ideas and discuss potential questions.
 
 Once you've [signed up][sign-up] all you will need to join us is a device that can watch a Youtube Livestream.
```

Comment:
Maybe worth re-working this sentence so that it flows a bit better given the new context above?
Perhaps something along the lines of "If you'd like to take part, there's still time to [sign up]"?

Reply:
https://github.com/srobo/website/pull/233/commits/2725ef0e22b25bbdf853dde9cc43170e7a36b9d6